packages/drawer/src/Default/Default.tsx

```diff
@@ -1,21 +1,15 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import { ThemeProvider, Offcanvas } from 'react-bootstrap';
-import { Component as Button } from '../../../button/src/Default/Default';
-import { DrawerDefaultProps, themePrefixesProps } from './Default.types';
+import { DrawerDefaultProps } from './Default.types';
 import * as styleMod from './styles.module.scss';
-import { createLocalProps } from '@owlui/utils';
```

Comment:
Imported `themePrefixesProps` from Utils to avoid the warning on build saying that we have duplicated exports.